indico/modules/events/editing/schemas.py

```diff
@@ -363,3 +380,10 @@ class Meta:
         rh_context = ('event',)
 
     principals = PrincipalList(many=True, allow_event_roles=True, allow_category_roles=True)
+
+
+class ServiceReviewEditableSchema(mm.Schema):
+    publish = fields.Boolean(default=True)
+    comment = fields.String()
+    comments = fields.List(fields.String())
```

Comment:
See my other comment:

- this is currently not taken into account
- we should probably accept the same data structure here like the one we'd POST to the comments endpoint